eticaPlanner/src/main/java/com/eticaplanner/eticaPlanner/user/UserController.java

```diff
@@ -19,22 +20,32 @@ public String SignUpView(Model model){
         model.addAttribute("viewName", "User/signUp");
         return "template/layout";
     }
-    
-    // 로그인
+
+    /**
+     * 로그인 화면
+     * @param model
+     * @return
+     */
     @GetMapping("/sign-in-view")
     public String SignInView(Model model){
         model.addAttribute("viewName", "User/signIn");
         return "template/layout";
     }
 
-    /* 추후 구현 예정
-    // 로그아웃
-    @GetMapping("/sign-out")
-    public String SignOutV(){
+    /**
+     * 로그아웃 API
+     * @param session
+     * @return
+     */
+    @RequestMapping("/sign-out")
+    public String SignOut(HttpSession session){
```

Comment:
```suggestion
    public String signOut(HttpSession session){
```
메서드는 소문자로 시작하는게 좋을거같아요